ad-fmcjesdadc1-ebz/ad_fmcjesdadc1_ebz.c

```diff
@@ -88,9 +88,29 @@ int main(void)
 	rx_xfer_1.start_address = XPAR_DDR_MEM_BASEADDR + 0x900000;
 #endif
 
+#ifdef ZYNQ_PS7
```

Comment:
we should try to keep these compiler switches to a minimum. If someting can be done without them then use that option. I'm not sure in this case what could be done.